libparsec/crates/protocol/tests/block.rs

```diff
@@ -2,7 +2,7 @@
 
 use hex_literal::hex;
 
-use libparsec_protocol::authenticated_cmds::v2 as authenticated_cmds;
+use libparsec_protocol::authenticated_cmds::v3 as authenticated_cmds;
```

Comment:
use `latest` instead of a specific version of the api, this will save us modifications when we will introduce new

(as a matter of fact, we want to test all version but this is another story: https://github.com/Scille/parsec-cloud/issues/4545 ^^)

Reply:
If I am not mistaken with `latest` we will be testing api v4, so the tests will also need to be updated whenever there is a difference between v3 and v4. Is this what you are suggesting?

Reply:
yes, APIv4 is considered the base (so all tests are against it), and additional tests should be provided to test specific stuff on APIv3

And in the long term, we want instead to have a proper set of test of each API family (with a way to factorize the tests, given a new API most likely introduce few breaking changes and we don't want to duplicate 1000s of line of tests ^^)

Reply:
Actually there are some significant changes between v3 and v4, including some semantic(?) changes. 

For example the `wait` parameter in `authenticated_cmds/events_listen` does no longer exists because : https://github.com/Scille/parsec-cloud/blob/cd6951efc537fabe65568207c8599e7b461dfed3/libparsec/crates/protocol/schema/authenticated_cmds/events_listen.json5#L8

I propose:
- To keep `v3` in tests to achieve the main goal of this PR (to drop API v2)
- Open another issue to "migrate" protocol tests from `v3` to `latest`.

If you are OK with this, then this PR can be re-reviewed and (hopefully!) merged.

Reply:
yeah I'm good with that ^^